templates/web/base/maps/openlayers.html

```diff
@@ -33,4 +33,6 @@
     [% IF map.copyright %]
     <div class="olControlAttribution" style="position: absolute;">[% map.copyright %]</div>
     [% END %]
-
+    [% IF c.user && c.user.is_inspector %]
+    <div id="fms_shortlist_all" class="hidden">Shortlist all visible</div>
```

Comment:
I guess it's to make it a bit more visible. The `sub_maps_links` part looks more related to map controls, whereas this is a specific action on reports.